requirements.txt

```diff
@@ -12,4 +12,5 @@ coloredlogs==10.*
 detect_delimiter==0.1.*
 pyyaml==5.*
 black==19.10.b0
-flatten-json==0.1.*
\ No newline at end of file
+flatten-json==0.1.*
+bump2version
```

Comment:
Can we please add bump2version and while we are at it also black to a requirements.dev.txt file. They are not required to use LUSID Python Tools only if you are developing.

Reply:
If you crate the new file, you can inherit the main requirements.txt by adding a line with “-r requirements.txt”

Reply:
Good idea. That's done in latest commit.